packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -105,6 +105,16 @@ exports.onCreateNode = ({ node, getNode, actions }, pluginOptions) => {
         Boolean(node.frontmatter.excludeFromSearchIndex) ||
         Boolean(pluginOptions.excludeFromSearchIndex),
     });
+    // Release note flag, used for filter in release notes query
+    actions.createNodeField({
+      node,
+      name: 'isReleaseNote',
```

Comment:
Why do we need this flag?

Reply:
I wrote why it is needed in the comment but to clarify:

- the release notes overview needs list of release notes, so this field enables filtering out only release notes when the query is executed.

Reply:
But why. You can use the graphql filters and for example filter by the file path or the name you gave to the source-filesystem.

I don't think we need to add a new field just for filtering.

Reply:
I thought about hose options but I could not find an implementation that works.